src/mlpack/methods/ann/activation_functions/softplus_function.hpp

```diff
@@ -0,0 +1,114 @@
+/**
+ * @file softplus_function.hpp
+ * @author Vivek Pal
+ *
+ * Definition and implementation of the softplus function.
+ *
```

Comment:
Sure. I found the citation [here](http://www.iro.umontreal.ca/~lisa/publications2/index.php/export/publication/67/bibtex). Is that format correct? I think may be "abstract" field can be dropped from it for our purposes.